Readme.md

```diff
@@ -1,63 +1,87 @@
 OhlohUI
 =======
 
+Dependencies:
+----------------
+
+* OhlohUI uses the ruby version 2.2.5. Please install ruby 2.
+* OhlohUI uses the postgresql database. Please install postgresql and create a new user on it.
+
 Getting Started:
 ----------------
 
-* Install Vagrant
-* Clone this repository
-* **`cd vagrant`**
-* **`vagrant up`**
+```
+$ git clone git@github.com:blackducksoftware/ohloh-ui.git
+$ cd ohloh-ui
+$ gem install bundler
+$ bundle install
+```
 
-The initial provisioning of the virtual machine will take roughly five minutes.
+The OhlohUI data is split between two databases in production. The development setup needs to reflect the same.
+The database names are configured in a file specific to each environment. For development, create a file **env.development**, with the following contents.
 
-After the inital sync, you need only run **`vagrant provision`** as that will
-run bundle install and restart the server for you. That should take just a few seconds.
+```
+DB_HOST = localhost
+DB_NAME =
+DB_USERNAME =
+DB_PASSWORD =
+
+FOREIGN_DB_HOST = localhost
+FOREIGN_DB_NAME =
+FOREIGN_DB_USERNAME =
+FOREIGN_DB_PASSWORD =
+
+TEST_DB_HOST = localhost
+TEST_DB_NAME =
+TEST_DB_USERNAME =
+TEST_DB_PASSWORD =
+```
```

Comment:
@drubio1989  and I found that we needed a set of `FOREIGN_TEST_DB_*` environment variables so that `rake db:drop` doesn't try to drop development or staging SecondBase database.  And that the `database.yml` file needed to reference these `FOREIGN_TEST_DB` in the `test:` section for SecondBase to use these environment variables.